datafusion/datasource-parquet/src/source.rs

```diff
@@ -559,24 +556,8 @@ impl FileSource for ParquetSource {
                     .predicate()
                     .map(|p| format!(", predicate={p}"))
                     .unwrap_or_default();
-                let pruning_predicate_string = self
-                    .pruning_predicate()
-                    .map(|pre| {
-                        let mut guarantees = pre
-                            .literal_guarantees()
-                            .iter()
-                            .map(|item| format!("{}", item))
-                            .collect_vec();
-                        guarantees.sort();
-                        format!(
-                            ", pruning_predicate={}, required_guarantees=[{}]",
-                            pre.predicate_expr(),
-                            guarantees.join(", ")
-                        )
-                    })
-                    .unwrap_or_default();
 
-                write!(f, "{}{}", predicate_string, pruning_predicate_string)
```

Comment:
This is what's causing tests to fail. Tests assert against the formatting of a ParquetSource and accessing it's pruning predicate method.